pages/settings/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-0 pb-4">
+    <v-row class="pt-3">
+      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-3 pb-4">
```

Comment:
Can't we put the v-col top padding to 4? I'm just nitpicking though :p

Reply:
Done. 4 is even :)